Core/src/core/smallEvents/bonusGuildPVEIsland.ts

```diff
@@ -53,7 +53,7 @@ async function applyPossibility(
 	};
 	if (issue === SmallEventBonusGuildPVEIslandResultType.SUCCESS && player.hasAGuild()) {
 		const guild = await Guilds.getById(player.guildId);
-		const caller = result.isExperienceGain ? guild.addExperience : guild.addScore;
+		const caller = result.isExperienceGain ? guild.addExperience.bind(guild) : guild.addScore.bind(guild);
```

Comment:
```suggestion
		const caller = (result.isExperienceGain ? guild.addExperience : guild.addScore).bind(guild);
```

Est ce que ca ca marche ?

Reply:
Oui